Makefile

```diff
@@ -3,7 +3,7 @@
 # The following line prefers `python` as this will allow this to work
 # in CI (e.g. Travis CI) and put up the configured Python version:
 # SYSTEMPYTHON = `which python python3 python2 | head -n 1`
-SYSTEMPYTHON = `which python2 python | head -n 1`
+SYSTEMPYTHON = `which python2 python python2.7 | head -n 1`
```

Comment:
I think it's important that we pick up `python2.7` if there happens to be a python3 `python` on the system, could you please re-order these to be `python2 python2.7 python`?